components/core/src/os/process/windows.rs

```diff
@@ -27,6 +28,10 @@ use super::{HabExitStatus, ExitStatusExt};
 
 const STILL_ACTIVE: u32 = 259;
 
+const PROCESS_EXITED: i32 = 0;
```

Comment:
Yeah I thought about the enum but that did not mesh with the Linux side which uses the libc signal constants.

Thinking again about this maybe a better approach would be to define an enum in mod.rs that just signifies graceful or abrupt shutdown.